pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.black]
+skip-string-normalization = true
+
+[tool.flake8]
+show-source = true
+max-line-length = 99
+application-import-names = "wazo_dird"
+exclude = [
+    "build",
+    ".tox",
+    ".eggs",
+    "integration_tests/assets/scripts"
+]
+
+ignore = [
+    "E203", # whitespace before ':' warnings
+    "E501", # line too long
+    "W503", # line break before binary operator
+]
+
+[tool.mypy]
+python_version = "3.10"
+
+show_error_codes = true
+explicit_package_bases = true
+check_untyped_defs = true
+warn_unused_configs = true
+follow_imports = "skip"
+ignore_missing_imports = true
+
+plugins = [
+    "sqlmypy"
+]
+exclude = [
+    "build",
+    "integration_tests/docker/broken-plugins/setup.py"
+]
```

Comment:
```suggestion
```

Reply:
I'd want to keep the exclude here too, because pre-commit is not the only way to use mypy. Editor integrations and manually running mypy(e.g. for more hands-on debugging or playing around with mypy options without the pre-commit layer) should benefit from the config too.

Reply:
I suppose... I just hate redundancy. I don't see why it would be necessary to run it another way. I agree it would be better if we could centralize it here, but it doesn't seem to work.

Reply:
I don't love redundancy, but I prefer flexibility in this case.